assets/css/sensei-modal.scss

```diff
@@ -0,0 +1,46 @@
+.sensei-course-theme__frame, .sensei-course-theme--focus-mode--animated, * {
+
+	[data-sensei-modal-overlay] {
+		position: fixed;
+		z-index: 1000;
```

Comment:
Have a small z-index issue, caused by the sidebar/main content relation when this is in the sidebar. Adding a `z-index: 10` to `.sensei-course-theme__sidebar` should fix it.

<img width="1687" alt="image" src="https://user-images.githubusercontent.com/176949/145284996-99f4351e-1dba-45b6-ae8c-f0c8cadb47d0.png">

Reply:
Nice catch! Fixed just like you suggested. d419c23f1cd2f2e90bc8a35a79c981c3a80aaa57

Reply:
Oh to also overlay the header I think we'll need 100+ on the z-index, my bad.

Reply:
You mean setting `z-index: 100+` on the header? The `100+` is not a valid value for `z-index`, so it simply removes it. Which solves the overlay problem, but now we have some content getting on top of header. See video below.

https://user-images.githubusercontent.com/2578542/145528018-9f588259-59c9-41f7-8fe7-698309b18d89.mp4

TIL about the [CSS stacking  context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context). In order to make sure the modal is stacked on top of all other elements, it needs to be rendered at the end of the body element.

So I update the `sensei-modal.js` to do it like that. Now `z-index` of the modal should not be a problem. 1e9d6cb575a197c224653df534bdd920806896fe

Reply:
Oh sorry I meant a `z-index` above `100`, for the sidebar (so it's above the header, since yeah, because of the stacking context, if its parent container is below something it will also be below it). 
But moving it out of the containers also works 👍